PrestaShop/PrestaShop
# Display "No picture" image for products with no image

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | You can add a "No-picture" image , Langage , it was not anymore displayed on 1.7 when a product have no image.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | See below please

------------
How to test ?
- First upload a "No-picture" image fraom admin > your langage ( no one is delivered on default ps instal) 
- Créate a product whithout any pictures,
- go on your front Office and see the result

------------
Before this commit, whe add :
![correctif_images_before_1](https://user-images.githubusercontent.com/3170104/35032556-8c98223a-fb67-11e7-82ed-d2d1c21cad26.png)
![correctif_images_before_2](https://user-images.githubusercontent.com/3170104/35032557-8caf3312-fb67-11e7-93fe-14748b397bca.png)


------------
After this commit, we have  Preston as "No-picture" image:
![correction_no_image_after_1](https://user-images.githubusercontent.com/3170104/35032576-9914d274-fb67-11e7-9bb0-ef177d766a74.png)
![correction_no_image_after_2](https://user-images.githubusercontent.com/3170104/35032577-992f45c8-fb67-11e7-861f-ea094a846d2f.png)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/8678)
